build.sbt

```
@@ -6,7 +6,11 @@ val catsV = "2.7.0"
 val scalacheckV = "1.15.4"
 val disciplineMunitV = "1.0.9"
 
-enablePlugins(SonatypeCiReleasePlugin)
+enablePlugins(TypelevelPlugin)
```

```suggestion
enablePlugins(TypelevelCiReleasePlugin)
```

We could also make this an auto-plugin.

---

Yeah, that's confusing.  I thought sbt-typelevel included all the doodads.